### PR TITLE
CANS-1192: Test reports

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

**Comment by DmitriyRudenko** [comment]
retest this please

File: docker-tests/entrypoint.sh [hunk 1]
```diff
@@ -42,8 +42,9 @@ if [ "$TEST_TYPE" == "smoke" ] || [ "$TEST_TYPE" == "functional" ]; then
   echo "PERRY_LOGIN_FORM_URL = '$PERRY_LOGIN_FORM_URL'"
   echo "TEST_TYPE = '$TEST_TYPE'"
   echo "TEST_CLASS = '$TEST_CLASS'"
-
-  java ${JAVA_OPT} -Dapi.url="${CANS_API_URL}" -Dperry.url="${PERRY_URL}" -Dlogin.form.target.url="${PERRY_LOGIN_FORM_URL}" -cp /opt/cans-api-test/resources:cans-api-test.jar org.junit.runner.JUnitCore ${TEST_CLASS}
+  pwd
+  cd /opt/cans-api-test
+  /opt/cans-api-test/gradlew clean :cansApiTests
```
Comment (SashaKyz) [inline docker-tests/entrypoint.sh]: Can you please use exit $?

File: docker-compose.yml [hunk 2]
```diff
@@ -62,6 +62,9 @@ services:
     image: cwds/cans-api-test
     hostname: cans-api-test
     entrypoint: sleep infinity
+    volumes:
```
Comment (denys-davydov) [inline docker-compose.yml]: Does this file work locally with this update?

Comment (DmitriyRudenko) [reply docker-compose.yml]: yes, It'll just create dirs

**Comment by DmitriyRudenko** [comment]
retest this please

**Comment by LeonidMarushevskyi** [comment]
Jenkins retest this please

**Pull request merged**